cdapython.py

```diff
@@ -63,6 +65,17 @@ def __init__(self, *args) -> None:
         self.query.node_type = _op
         self.query.l = _l
         self.query.r = _r
+       
+    @staticmethod
+    def sql(sql, host=CDA_API_URL, dry_run=False, offset=0, limit=1000):
+        with ApiClient(
+            configuration= Configuration(host=host)
+        ) as api_client:
+            api_instance = QueryApi(api_client)
+            api_response = api_instance.sql_query(sql)
+        if dry_run is True:
```

Comment:
Could be replaced with `if dry_run:`

Reply:
You are right about that thanks i did miss that